ui/raidboss/data/05-shb/raid/e9s.js

```diff
@@ -78,17 +118,8 @@ export default {
           return 8;
         return 4;
       },
-      alertText: (data, _, output) => output.text(),
-      outputStrings: {
-        text: {
-          en: 'Protean',
-          de: 'Himmelsrichtungen',
-          fr: 'Position',
-          ja: '散開',
-          cn: '散开',
-          ko: '산개',
-        },
-      },
+      alertText: (data, _, output) => output['protean'](),
+      outputStrings: artOfDarknessOutputStrings,
```

Comment:
Returning your nit from wolex, but using all the output strings here means that the ui will have options for all of these, which is kind of awkward.

```suggestion
      alertText: (data, _, output) => output.text(),
      outputStrings: { text: artOfDarknessOutputStrings.protean },
```

Here, and elsewhere.